securedrop_client/gui/widgets.py

```diff
@@ -1874,17 +1875,24 @@ def _on_export_clicked(self):
             return
 
         dialog = ExportDialog(self.controller, self.file.uuid)
-        # The underlying function of the `export` method makes a blocking call that can potentially
-        # take a long time to run (if the Export VM is not already running and needs to start, this
-        # can take 15 or more seconds). Calling `QApplication.processEvents` ensures that the `show`
-        # event is processed before the blocking call so that the user can see the dialog with a
-        # message to wait before the blocking call. We also call `exec` afterwards in order to give
-        # control to the dialog for the rest of the export process.
         dialog.show()
-        QApplication.processEvents()
```

Comment:
thanks for removing, confirmed that dialog.export() and dialog.print() here are calling out to the controller object which will emit a signal to the export thread to print/export to USB